plugins/generic/externalFeed/controllers/grid/ExternalFeedGridCellProvider.inc.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @file controllers/grid/ExternalFeedGridCellProvider.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ExternalFeedGridCellProvider
+ * @ingroup controllers_grid_externalFeed
+ *
+ * @brief Class for a cell provider to display information about external feed
+ */
+
+import('lib.pkp.classes.controllers.grid.GridCellProvider');
+import('lib.pkp.classes.linkAction.request.RedirectAction');
+
+class ExternalFeedGridCellProvider extends GridCellProvider {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		parent::__construct();
+	}
+	
+	//
+	// Template methods from GridCellProvider
+	//
+	/**
+	* Get cell actions associated with this row/column combination
+	* @param $row GridRow
+	* @param $column GridColumn
+	* @return array an array of LinkAction instances
+	*/
+	function getCellActions($request, $row, $column, $position = GRID_ACTION_POSITION_DEFAULT) {

```

Comment:
I think you can remove this entire implementation, since it just falls back on the parent implementation.